src/Chainweb/Miner/Coordinator.hs

```diff
@@ -38,6 +38,7 @@ module Chainweb.Miner.Coordinator
 -- ** Internal Functions
 , newWork
 , publish
+, removePrimed
```

Comment:
Why export?

Reply:
Also, why export `newWork`? Clearly nothing broke by changing its signature.

Reply:
Oh, I used `removePrimed` for some ad-hoc testing/debugging. Forgot to remove it.

```suggestion
```

Reply:
Not sure about `newWork`. It might be used in test code. I'll double check.